db/migrate/20230328152627_add_users_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :users, null: false, foreign_key: true
```

Comment:
- [Optional] I think it's preferable to stick to either user_id or author_id as a reference key though differences arise in the ERD